interactions/client/bot.py

```diff
@@ -363,6 +365,55 @@ async def change_presence(self, presence: ClientPresence) -> None:
         """
         await self._websocket._update_presence(presence)
 
+    async def get(
+        self, model: T, http_method: Optional[str] = None, **kwargs: Dict[str, Snowflake]
+    ) -> T:
+        r"""
+        A helper function for getting a model from the Discord API.
+
+        :param model: The model to get.
+        :type model: object
+        :param http_method: The HTTP method to use instead of `get_x`.
+        :type http_method: Optional[str]
+        :param \**kwargs: The arguments to pass to the HTTP method.
+        :type \**kwargs: Dict[str, Snowflake]
+        :return: The model.
```

Comment:
```suggestion
        :return: The model we're trying to get.
```

This should have some clarification consistent to the rest of our documentation

Reply:
Done